docs/content/how-to/openstack/hostedcluster.md

```diff
@@ -0,0 +1,198 @@
+# Create a HostedCluster on OpenStack
+
+Once all the [prerequisites](prerequisites.md) are met, it is now possible to create a Hosted Cluster on OpenStack.
+
+Here are the available options specific to the OpenStack platform:
+
+| Option                           | Description                                                                                  | Required | Default       |
+|----------------------------------|----------------------------------------------------------------------------------------------|----------|---------------|
+| `--openstack-ca-cert-file`       | Path to the OpenStack CA certificate file                                                    | No       |               |
+| `--openstack-cloud`              | Name of the cloud in `clouds.yaml`                                                           | No       | `'openstack'` |
+| `--openstack-credentials-file`   | Path to the OpenStack credentials file                                                       | No       |               |
+| `--openstack-external-network-id`| ID of the OpenStack external network                                                         | No       |               |
+| `--openstack-ingress-floating-ip`| A floating IP for OpenShift ingress                                                          | No       |               |
+| `--openstack-node-additional-port`| Attach additional ports to nodes. Params: `network-id`, `vnic-type`, `port-security`, `address-pairs`. | No       |               |
+| `--openstack-node-availability-zone`| Availability zone for the nodepool                                                        | No       |               |
+| `--openstack-node-flavor`        | Flavor for the nodepool                                                                      | Yes      |               |
+| `--openstack-node-image-name`    | Image name for the nodepool                                                                  | Yes      |               |
+
+Below is an example of how to create a cluster using environment variables and the `hcp` cli tool.
+
+!!! note
+
+    The --release-image flag could be used to provision the HostedCluster with a specific OpenShift Release (the hypershift operator has a support matrix of releases supported by a given version of the operator)
```

Comment:
```suggestion
    The `--release-image` flag could be used to provision the HostedCluster with a specific OpenShift Release (the hypershift operator has a support matrix of releases supported by a given version of the operator)
```

I thought this opt was only for the `hypershift` binary...

Reply:
```shell
Usage:
  hcp create cluster openstack [flags]

Flags:
(...)

Global Flags:
(...)
      --release-image string                        The OCP release image for the cluster